examples/modular-drivers/Cargo.toml

```diff
@@ -1,16 +1,19 @@
 [package]
-name = "micro-rdk-esp32-sensor-examples"
-resolver = "2"
-version = "0.2.0"
-edition = "2021"
-authors = ["Viam, inc. <contact@viam.com>"]
-resolver = "2"
-rust-version = "1.75"
-license = "AGPL-3.0"
+name = "micro-rdk-modular-driver-example"
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[features]
+esp32 = []
```

Comment:
I think enabling the `esp32` feature her should enable it in `micro-rdk` as well. Something like:
```
[features]
esp32 = ['micro-rdk/esp32']
```